ssm_endpoint_sg.tf

```
@@ -0,0 +1,23 @@
+# Security group for the SSM interface endpoint (and other endpoints
+# required when using the SSM service) in the private subnet
+resource "aws_security_group" "ssm_endpoint" {
+  provider = aws.sharedservicesprovisionaccount
+
+  vpc_id = aws_vpc.the_vpc.id
+
+  tags = {
+    Name = "SSM endpoint"
```

So I know I questioned you about this previously and you changed it to singular. However upon revisiting this and checking against `vpc_endpoints.tf` I see that this security group is applied to multiple VPC endpoints. I believe this should be plural to be consistent with the other SGs applied to multiple VPC endpoints but please let me know if I'm reading this incorrectly.

```suggestion
    Name = "SSM endpoints"
```

---

Ugh.  Please see commit fa1e368.

---

Apologies and thank you for your patience 😅